SEC/Equities/EquityInstruments.rdf

```diff
@@ -572,45 +667,29 @@
 		<fibo-fnd-utl-av:synonym>preference share</fibo-fnd-utl-av:synonym>
 	</owl:Class>
 	
-	<owl:Class rdf:about="&fibo-sec-eq-eq;PreferredShareCoupon">
-		<rdfs:subClassOf rdf:resource="&fibo-fnd-agr-agr;Commitment"/>
-		<rdfs:subClassOf>
-			<owl:Restriction>
-				<owl:onProperty rdf:resource="&fibo-fnd-pas-psch;hasPaymentAmount"/>
-				<owl:onClass rdf:resource="&fibo-fnd-acc-cur;MonetaryAmount"/>
-				<owl:maxQualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">1</owl:maxQualifiedCardinality>
-			</owl:Restriction>
-		</rdfs:subClassOf>
-		<rdfs:subClassOf>
-			<owl:Restriction>
-				<owl:onProperty rdf:resource="&fibo-sec-eq-eq;hasDividendRate"/>
-				<owl:onClass rdf:resource="&fibo-fnd-acc-cur;PercentageMonetaryAmount"/>
-				<owl:maxQualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">1</owl:maxQualifiedCardinality>
-			</owl:Restriction>
-		</rdfs:subClassOf>
-		<rdfs:subClassOf>
-			<owl:Restriction>
-				<owl:onProperty rdf:resource="&fibo-sec-eq-eq;hasGracePeriod"/>
-				<owl:onClass rdf:resource="&fibo-fnd-dt-fd;Duration"/>
-				<owl:maxQualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">1</owl:maxQualifiedCardinality>
-			</owl:Restriction>
-		</rdfs:subClassOf>
+	<owl:Class rdf:about="&fibo-sec-eq-eq;PreferredShareWithAdjustableRateDividend">
+		<rdfs:subClassOf rdf:resource="&fibo-sec-eq-eq;PreferredShare"/>
 		<rdfs:subClassOf>
 			<owl:Restriction>
-				<owl:onProperty rdf:resource="&fibo-sec-eq-eq;hasDistributionMethod"/>
-				<owl:onClass rdf:resource="&fibo-sec-eq-eq;DividendDistributionMethod"/>
-				<owl:maxQualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">1</owl:maxQualifiedCardinality>
+				<owl:onProperty rdf:resource="&fibo-sec-eq-eq;hasDividend"/>
+				<owl:someValuesFrom rdf:resource="&fibo-sec-eq-eq;AdjustableRateDividend"/>
 			</owl:Restriction>
 		</rdfs:subClassOf>
+		<rdfs:label>preferred share with adjustable rate dividend</rdfs:label>
+		<owl:disjointWith rdf:resource="&fibo-sec-eq-eq;PreferredShareWithFixedRateDividend"/>
+		<skos:definition>preferred share whose dividends will vary with a benchmark, most often a T-bill rate</skos:definition>
```

Comment:
Need a property to reference the Benchmark; and there will presumably be an adjustment e.g. +0.5%

Reply:
Those properties are on the adjustable rate dividend itself, rather than on the share.

Reply:
Did you just add them?> My earlier comment was that I could not see such properties.

Reply:
Raised #1138